### PR TITLE
Add New Methods: `subDay`, `getLastWeek`, `addDay`, `getLastMonth`

### PR DESCRIPTION
**Pull request by majidalaeinia**
- `subDay`
- `getLastWeek`
- `addDay`
- `getLastMonth`

### CODE REVIEW

File: src/Jalalian.php [hunk 1]
```diff
@@ -648,16 +653,31 @@ public function getNextWeek(): Jalalian
...
+    }
+
     public function addDays(int $days = 1): Jalalian
     {
         return static::fromCarbon($this->toCarbon()->addDays($days));
     }
 
+    public function addDay(): Jalalian
+    {
+        return static::fromCarbon($this->toCarbon()->addDays());
```
Comment (morilog) [inline src/Jalalian.php:668]: use `addDays(1)` here

File: src/Jalalian.php [hunk 2]
```diff
@@ -354,6 +354,11 @@ public function subDays(int $days = 1): Jalalian
         return static::fromCarbon($this->toCarbon()->subDays($days));
     }
 
+    public function subDay(): Jalalian
+    {
+        return static::fromCarbon($this->toCarbon()->subDays(1));
```
Comment (morilog) [inline src/Jalalian.php:359]: use `subDays(1)` here

File: src/Jalalian.php (hunk 1 above)
Comment (majidalaeinia) [reply src/Jalalian.php:668]: I see, I wanted it to be like `Carbon` and support the same name, but I can change it.

File: src/Jalalian.php (hunk 2 above)
Comment (majidalaeinia) [reply src/Jalalian.php:359]: I see, I wanted it to be like Carbon and support the same name, but I can change it.

File: src/Jalalian.php (hunk 1 above)
Comment (morilog) [reply src/Jalalian.php:668]: No. I mean use `addDays(1)` instead of `static::fromCarbon($this->toCarbon()->addDays());`

Comment (majidalaeinia) [reply src/Jalalian.php]: Done.

File: src/Jalalian.php (hunk 2 above)
Comment (majidalaeinia) [reply src/Jalalian.php]: Done.

**Pull request merged**